e2e-tests/demand-backup/conf/restore-on-demand-backup-pvc-2.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBClusterRestore
+metadata:
+  name: on-demand-backup-pvc-2
+spec:
+  pxcCluster: some-name
+  backupSource:
+    destination: pvc/some-name-xb-on-demand-backup-pvc
```

I think there should be backupsource of second PVC
```suggestion
    destination: pvc/some-name-xb-on-demand-backup-pvc-2
```

---

https://github.com/percona/percona-xtradb-cluster-operator/pull/556/commits/edfe5858f06e192094e7703e3ba62b0afea5647e